{{cookiecutter.project_slug}}/backend/app/app/crud/base.py

```diff
@@ -41,7 +41,7 @@ def update(
         self, db_session: Session, *, db_obj: ModelType, obj_in: UpdateSchemaType
     ) -> ModelType:
         obj_data = jsonable_encoder(db_obj)
-        update_data = obj_in.dict(skip_defaults=True)
+        update_data = obj_in.dict(exclude_unset=True)
```

Comment:
This will conflict with [PR #106](https://github.com/tiangolo/full-stack-fastapi-postgresql/pull/106/files#diff-ace326c3ae9df4eb050c95eade5d86d9R42)